js/.eslintrc

```diff
@@ -0,0 +1,10 @@
+{
+    "root": true,
+    "parser": "@typescript-eslint/parser",
+    "plugins": ["@typescript-eslint", "require-extensions", "simple-import-sort"],
+    "extends": ["eslint:recommended", "plugin:@typescript-eslint/recommended", "plugin:require-extensions/recommended"],
+    "rules": {
+        "@typescript-eslint/no-unused-vars": ["error", { "argsIgnorePattern": "^_" }],
+        "simple-import-sort/imports": "error"
```

Comment:
Must add `rules-of-hooks` here, which would have caught that dependency related bug in `useSeedVault.ts`.

https://www.npmjs.com/package/eslint-plugin-react-hooks